src/App.tsx

```diff
@@ -1,9 +1,34 @@
-function App() {
+import PanelsWrapper from './components/PanelsWrapper';
+import { useEffect, useState } from 'react';
+import { useConfig} from '@/contexts/ConfigContext'
+
+
+
+function App({customConfig}) {
```

Comment:
Type?

Reply:
converted the component to FC component and used a typescript interface for its type.